torchrl/data/tensordict/tensordict.py

```diff
@@ -1231,6 +1302,9 @@ def __iter__(self) -> Generator:
         for i in range(length):
             yield self[i]
 
+    def flatten_keys(self, separator=",", inplace: bool = True) -> _TensorDict:
+        return flatten_keys(self, separator=separator, inplace=inplace)
```

Comment:
Maybe its better to mark `flatten_keys` as a static method. The current implementation (while correct) looks a bit confusing

Reply:
ok i removed the function and made it a method. 
not sure we can make it static as we need the `self` arg